### PR TITLE
Update fastify to v3

### PR DESCRIPTION
**Pull request by frikille**
* Updated fastify to v3
* Replaced greenkeeper config with dependabot
* Moved CI from Travis to Github actions

This PR unblocks fastify/fastify#2217

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: CI workflow
```
Comment (Eomm) [inline .github/workflows/ci.yml:1]: Could you update the readme accordingly?

```
![CI workflow](https://github.com/fastify/fastify-env/workflows/CI%20workflow/badge.svg)
```

Comment (frikille) [reply .github/workflows/ci.yml:1]: Sure, updated

File: index.js [hunk 2]
```diff
@@ -15,6 +15,6 @@ function loadAndValidateEnvironment (fastify, opts, done) {
 }
 
 module.exports = fp(loadAndValidateEnvironment, {
-  fatify: '>=0.39.0',
+  fastify: '>=0.39.0',
```
Comment (mcollina) [inline index.js:18]: make it 3.x, we are bumping a major due to fastify-plugin

File: .dependabot/config.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+version: 1
+update_conigs:
+  package_manager: "javascript"
+  directory: "/"
+  update_schedule: "daily"
```
Comment (mcollina) [inline .dependabot/config.yml:5]: Can you fix this? Check the other projects.

File: index.js [hunk 4]
```diff
@@ -15,6 +15,6 @@ function loadAndValidateEnvironment (fastify, opts, done) {
 }
 
 module.exports = fp(loadAndValidateEnvironment, {
-  fatify: '>=0.39.0',
+  fastify: '>=3.x',
```
Comment (mcollina) [inline index.js:18]:
```suggestion
  fastify: '3.x',
```

Comment (Eomm) [reply index.js]: Why we removed the `>=` ?

**Pull request merged**